utils/calibration_utils.py

```diff
@@ -0,0 +1,32 @@
+from contextlib import contextmanager
+import time
+
+
+def set_calibration_file(channel_access, filename):
+    """
+    Sets a calibration file. Retries if it didn't set properly first time.
+    """
+    max_retries = 10
+
+    for _ in range(max_retries):
+        channel_access.set_pv_value("CAL:SEL", filename)
```

Comment:
I think "CAL:SEL" should be moved out into a constant here in case we ever change the pv name